.travis.rosinstall.jade

```diff
@@ -0,0 +1,20 @@
+- git:
```

Comment:
I don't like copies ;)
link it or set `ROSINSTALL_FILENAME=.travis.rosinstall.kinetic`..